lte/gateway/release/build-magma.sh

```diff
@@ -260,7 +260,7 @@ if [ -d ${PY_TMP_BUILD} ]; then
 fi
 
 FULL_VERSION=${VERSION}-$(date +%s)-${COMMIT_HASH}
-COMMIT_HASH_WITH_VERSION=${VERSION}-${COMMIT_HASH}
+COMMIT_HASH_WITH_VERSION="magma@${VERSION}.$(git rev-list --count HEAD)-$(git log --pretty=format:'%h' -n 1)"
```

Comment:
I just noticed that COMMIT_HASH is also given by a CLI parameter of this script. Are we sure that this is executed inside the repository so that the `git` commands will work?

Reply:
Yes, works as expected. The path with which the script is called, shows that it is called from inside the repository. https://github.com/magma/magma/blob/32a8a31ed4eb40cc8e0e0b0fc91ce8c0856ce05d/lte/gateway/fabfile.py#L125